### PR TITLE
Clarifed README setup steps.

### PR DESCRIPTION
**Pull request by magyarn**
I recently completed the setup steps for the `djangoproject` site and had to perform a few additional steps, which are included in this PR along with a few other suggestions for additional clarity.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -96,11 +110,18 @@ To run locally, do the usual:
...
    This is unnecessary with some browsers (e.g. Opera and Chromium/Chrome) as
    they handle localhost subdomains automatically.
 
+   To manually edit ``/etc/hosts`` on a Mac:
+       - enter ``sudo nano /etc/hosts``.
+       - use the arrow keys to update 127.0.0.1 to look like the above. 
+       - enter ``Cntrl + X`` and you will be prompted to save the file.
+       - enter ``Y`` to save it.
+       - hit Enter again to confirm the name of the file you want to save.
+
```
Comment (felixxm) [inline README.rst]: I don't think that such descriptive paragraph is necessary.

File: README.rst [hunk 2]
```diff
@@ -96,11 +110,18 @@ To run locally, do the usual:
    and ``dashboard.djangoproject.localhost`` hostnames with your ``/etc/hosts``
    file to ``localhost``/``127.0.0.1``::
 
-     127.0.0.1  docs.djangoproject.localhost www.djangoproject.localhost dashboard.djangoproject.localhost
+     127.0.0.1  localhost docs.djangoproject.localhost www.djangoproject.localhost dashboard.djangoproject.localhost
```
Comment (felixxm) [inline README.rst]: `127.0.0.1  localhost` should be already in the `/etc/hosts`. IMO we can clarify that it's an example of new line, e.g.
```
...
   and ``dashboard.djangoproject.localhost`` hostnames with your ``/etc/hosts``
   file to ``localhost``/``127.0.0.1`` by adding::
```

File: README.rst [hunk 3]
```diff
@@ -33,9 +35,14 @@ To run locally, do the usual:
...
-   to your ~/.bashrc (or ~/.zshrc if you're using zsh) file and then run
-   `source ~/.bashrc` (or `source ~/.zshrc`) to load the changes.
+#. Add `export DJANGOPROJECT_DATA_DIR=~/.djangoproject` (without the backticks)
+   to your ~/.bashrc (or ~/.zshrc if you're using zsh, ~/.bash_profile if you're 
+   on a Mac) file and then run `source ~/.bashrc` (or `source ~/.zshrc`, or 
+   `source ~/.bash_profile`) to load the changes.
+
+   If you're having trouble locating this file and you use a Mac, open your preferred 
+   text editor and go to `File > Open`. Then enter `Cmd + Shift + .` to show hidden
+   files. Locate `.bash_profile` and open it.
```
Comment (felixxm) [inline README.rst]: Please chop this sentence, I don't think that this tip is necessary.

File: README.rst [hunk 4]
```diff
@@ -33,9 +35,14 @@ To run locally, do the usual:
       "trac_db_host": "localhost",
       "trac_db_password": "secret" }
 
-   Add `export DJANGOPROJECT_DATA_DIR=~/.djangoproject` (without the backticks)
-   to your ~/.bashrc (or ~/.zshrc if you're using zsh) file and then run
-   `source ~/.bashrc` (or `source ~/.zshrc`) to load the changes.
+#. Add `export DJANGOPROJECT_DATA_DIR=~/.djangoproject` (without the backticks)
+   to your ~/.bashrc (or ~/.zshrc if you're using zsh, ~/.bash_profile if you're 
+   on a Mac) file and then run `source ~/.bashrc` (or `source ~/.zshrc`, or
```
Comment (felixxm) [inline README.rst]: `Mac` -> `Mac OS`

File: README.rst [hunk 5]
```diff
@@ -9,7 +9,9 @@ djangoproject.com source code
 
 To run locally, do the usual:
 
-#. Create a Python 3.5 virtualenv
+#. Create a Python 3.5 virtualenv::
+
+    python3 -m venv YOUR_VENV_DIRECTORY_NAME_HERE
```
Comment (felixxm) [inline README.rst]: It is not the only way, I would remove this.

File: README.rst [hunk 6]
```diff
@@ -71,7 +78,14 @@ To run locally, do the usual:
...
 
-#. For docs::
+#. For docs
+
+   Before running the next commands, make sure you have `gettext` installed.
+   If you don't, run the following::
+
+    brew install gettext
+
+   Then run::
```
Comment (felixxm) [inline README.rst]: Maybe:
```
#. For docs (next step requires ``gettext``)::

    ./manage.py loaddata doc_releases
    ./manage.py update_docs
```

File: README.rst (hunk 4 above)
Comment (schinckel) [reply README.rst]: > `Mac` -> `Mac OS`

_technically_ it's now `macOS` ;)

File: README.rst (hunk 2 above)
Comment (Crocmagnon) [reply README.rst]: I find the current sentence already pretty clear. It doesn’t hurt to add a new line with these domains.

File: README.rst (hunk 4 above)
Comment (Crocmagnon) [reply README.rst]: And I believe that the default interpreter on macOS Catalina has changed to zsh, so it’d be .zshrc on macOS too.

**Review by carltongibson — APPROVED** [review]
Hi @magyarn — thanks for this. I've rebased and pulled in @felixxm's comments. 👍

**Pull request merged**